corehq/apps/reports/filters/users.py

```diff
@@ -410,3 +429,75 @@ def get_user_toggle(request):
     elif group or individual:
         show_filter = False
     return toggle, show_filter
+
+
+class SubmitHistoryFilter(ExpandedMobileWorkerFilter):
+    options_url = 'emwf_options_all_users'
+
+    @property
+    @memoized
+    def utils(self):
+        return SubmitHistoryUtils(self.domain)
+
+    @property
+    def filter_context(self):
+        context = super(SubmitHistoryFilter, self).filter_context
+        url = reverse(self.options_url, args=[self.domain])
+        context.update({'endpoint': url})
+        if self.request.project.uses_locations:
+            context.update({'search_help_inline': self.search_help_inline})
+        return context
+
+    @classmethod
+    def user_es_query(cls, domain, mobile_user_and_group_slugs, request_user):
```

Comment:
This is a very large method to duplicate so much - it would be a pain to update both versions.  I think you can cheat a little here and just make the modifications to the base class.  If "DEACTIVATED" isn't an option, then it'll always be False anyways, and the behavior will be the same.